test/__tests__/data/kustomer_input.json

```diff
@@ -90,9 +90,7 @@
           "namespace": "com.rudderlabs.javascript"
         },
         "traits": {
-          "userId": "annodD",
           "name": "Ano Domeni",
-          "email": "ano@rudderstack.com",
```

Comment:
Is this negative test case ? removing an email.

Reply:
I removed it to basically allow an uncovered transformer flow to increase coverage, as we already had other tests with the email flow